src/app/public/modules/list-column-selector-action/list-column-selector-action.component.ts

```diff
@@ -56,17 +56,32 @@ import {
   SkyColumnSelectorComponent
 } from '../column-selector/column-selector-modal.component';
 
+/**
+ * Provides a column selector modal for a list grid view when placed in a
+ * [list toolbar](https://developer.blackbaud.com/skyux/components/list/toolbar).
+ */
 @Component({
   selector: 'sky-list-column-selector-action',
   templateUrl: './list-column-selector-action.component.html'
 })
 export class SkyListColumnSelectorActionComponent implements AfterContentInit {
+  /**
+   * Enables the column selector in the list toolbar. Set this attribute to the instance of
```

Comment:
The current docs have sections for these inputs and outputs at https://developer.blackbaud.com/skyux/components/list/grid#list-column-selector-action-properties and https://developer.blackbaud.com/skyux/components/list/grid#list-column-selector-action-events. However, `docs-tools` does not create tables for this content. Should it? Or do we need a separate topic for this component now?

Reply:
Definitely a bug. Filed: https://github.com/blackbaud/skyux-docs-tools/issues/125

Reply:
Should we resolve this conversation and rely on the `docs-tools` issue? Or keep it open as a reminder?

Reply:
Let's resolve it, get this PR merged, then once the issue is fixed all we need to do is update the dependencies. I've marked this in our spreadsheet as blocked for release so we won't forget.